feat(cloud-v2): Add external calls metrics (#385)
front-commerce/developers.front-commerce.com: source/docs/advanced/production-ready/metrics.md

You need two steps to enable this endpoint

> - `npm install express-prom-bundle prom-client --save`

Reviewer: Why the `--save` option? since version 5 this is automatic https://blog.npmjs.org/post/161081169345/v500
> [npm will --save by default now](https://twitter.com/maybekatz/status/859229741676625920). Additionally, package-lock.json will be automatically created unless an npm-shrinkwrap.json exists. ([#15666](https://github.com/npm/npm/pull/15666))

and v5 was last seen at node@8.11.4

Author: My mind does not wish to admit it -_- 
removed :+1:

- define the environment variable `FRONT_COMMERCE_CLOUD_METRICS_KEY` with a basic authentication token (base64 encoded, e.g. `dXNlcjpwYXNzd29yZA==` for `user:password`)

## Access it

Call `https://<your_website>/metrics` with the defined header `Authorization: Basic <FRONT_COMMERCE_CLOUD_METRICS_KEY>`

# Other metrics

## Measure external services call times

> Once the `/metrics` endpoint is enabled, you can count the time spent in calling external services by installing two more modules

Reviewer: ```suggestion
Once the `/metrics` endpoint is enabled, you can count the time spent calling external services by installing two additional modules
```

> - `npm i @opentelemetry/instrumentation-http @opentelemetry/instrumentation --save`

Reviewer: same for the `--save` as the previous review

> This will append the following kind of metrics to the `/metrics` endpoint

Reviewer: ```suggestion
This will append the following metrics to the `/metrics` endpoint
```
